init-odkx-sync-endpoint.py

```diff
@@ -32,11 +39,48 @@ def run_interactive_config():
     typer.echo("We'll need some information from you to get started though...")
     time.sleep(1)
     typer.echo("")
+
+    allow_default = typer.confirm("Do you wish to use default configuratilon?", default=True)
+    typer.echo("")
+
+    if not allow_default:
+        if is_cache_present():
+            allow_cache = typer.confirm("Do you wish to use cached configuratilon?", default=True)
+            typer.echo("")
+            if allow_cache:
+                cached_data = load_progress()
+                enforce_https = json.dumps(cached_data['enforce_https'])
```

Comment:
I think you can read the data directly from the cached_data object rather than needing to dump here, or is there a reason you want enforce_https and manual_certificate to be json strings instead of python objects?